apps/desktop/src/views/settings/index.tsx

```diff
@@ -26,8 +27,9 @@ export const SettingsView = ({ update }: { update: Update | null }) => {
           setCurrentTab(value);
         }}
       >
-        <TabsList className="grid w-full grid-cols-2">
+        <TabsList className="grid w-full grid-cols-3 rounded-t-none">
           <TabsTrigger value="account">General</TabsTrigger>
+          <TabsTrigger value="configuration">Configuration</TabsTrigger>
```

Comment:
```suggestion
          <TabsTrigger value="configuration">Customize</TabsTrigger>
```

This might be better, can we update the filename too?